src/ClaimService.Business/Features/Categories/Queries/GetCategories/CategoryInfo.cs

```diff
@@ -6,11 +6,26 @@ namespace LT.DigitalOffice.ClaimService.Business.Features.Categories.Queries.Get
 
 public class CategoryInfo
```

Comment:
this class is identical to CategoryResponse, is it really needed?

Reply:
for later